src/pages/ProjectsPage.js

```diff
@@ -1,126 +1,146 @@
-                      <LinkContainer to={`/projects/${selected}`}>
-                        <Button className="primary-button"> Work on this project! </Button>
-                      </LinkContainer>
-                    </div>
-                  </Body>
-                </Card>
-              </Col>
-            </Row>
-          </Container>
-        )}
-      </>
-    );
+// example projects, delete this later
+const example = (num) => ({
+  id: num,
+  isUrgent: num %2 === 0,
+  orgName: 'HackBeanpot ' + num,
+  orgAbout:
+    'this is what our org does. Which likely has a lot of words, resulting in multiple lines on the screen. We love beans in our org. bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean',
+  orgEmail: 'team@hbp.com',
+  orgPhone: null,
+  projectTitle: 'Sample Website Project ' + num,
+  projectAbout:
+    'This is what the project is about. Which is about beans. bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean',
+  projectTime: 'Estimated 2 weeks'
+})
+
+const sampleProjects = [example(1), example(2), example(3), example(4), example(5), example(6), example(7)];
+
+const ProjectCard = ({ project, isSelectedView, setSelected, selected }) => {
+  const { id, projectTitle, orgName, isUrgent, projectAbout } = project;
+  const getClass = () => {
+    let className = 'preview-card';
+    if (isSelectedView) {
+      className += ' select-view-preview-card'
+
+      if (selected === id) {
+        className += ' is-select-preview-card'
+      }
+    }
+    return className
   }
+
+  return (
+    <Card
+      className={getClass()}
+      onClick={() => setSelected(id)}>
+      <Body>
+        <h5> {projectTitle} </h5>
+        <Subtitle className="mb-2 preview-subtitle">{orgName}</Subtitle>
+        {isUrgent ?
+          <span className="project-card-urgent">Urgent</span> :
+          <span className="project-tag-space" />}
+        <p className="project-card-description">{projectAbout}</p>
+        <div className="project-more-link float-right">
+          More details...
+        </div>
+      </Body>
+    </Card>
+  )
+}
+
+const SelectedCard = ({ project, setSelected }) => {
+  const { id, orgName, orgAbout, projectTitle, projectAbout, projectTime, isUrgent } = project;
+
+  return (
+    <Card className="project-description-card">
+      <Body>
+        <div className="project-description-close float-right" onClick={() => setSelected(-1)} />
+
+        <h4> {projectTitle}</h4>
+        <Subtitle className="mb-2 project-description-subtitle">{orgName}</Subtitle>
+        {isUrgent && <div className="project-card-urgent">Urgent </div>}
+
+        <h5 className="pt-3"> About {orgName} </h5>
+        <p>{orgAbout}</p>
+
+        <h5 className="pt-3"> About the Project </h5>
+        <p>{projectAbout}</p>
+
+        <h5 className="pt-3"> Project Timeline </h5>
+        <p className="card-text">{projectTime}</p>
```

Comment:
We can change this to `{!!projectTime ? projectTime : 'Not Specified'}` since this is an optional field